docs/general/rendering.md

```diff
@@ -81,8 +81,12 @@ PR please :)
 
 #### meta
 
-If you want a `meta` attribute in your response, specify it in the `render`
-call:
+A `meta` member can be used to include non-standard meta-information. `meta` can
```

Comment:
You're right. I saw that later :embarrassed:

On Wed, Mar 9, 2016 at 8:40 AM Ben Mills <notifications@github.com> wrote:

> In docs/general/rendering.md
> <https://github.com/rails-api/active_model_serializers/pull/1566#discussion_r55526872>
> :
>
> > @@ -81,8 +81,12 @@ PR please :)
> >
> >  #### meta
> >
> > -If you want a `meta` attribute in your response, specify it in the `render`
> > -call:
> > +A `meta` member can be used to include non-standard meta-information. `meta` can
>
> Sure, although that's copy from the JSON API docs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1566/files#r55526872>
> .
>